### PR TITLE
fix order news

### PR DESCRIPTION
**Pull request by emnlpstr**
fixed ordering to news on home

## Checklist

- [x] I followed the indications in the [CONTRIBUTING](CONTRIBUTING.md)
- [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?
- [x] Ready for review! :rocket:

### CODE REVIEW

**Review by bfabio — COMMENTED** [review]
Is this to sort news in home page, depending on their status?

I think it'd be way more readable using [`Array.sort()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) with an explicit sort function.

fe:
```javascript
data.windows.reverse.sort((a, b) => {
  if (a.new > b.new) return 1;
  [...]
}
```

**Comment by emnlpstr** [comment]
> Is this to sort news in home page, depending on their status?
> 
> I think it'd be way more readable using [`Array.sort()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) with an explicit sort function.
> 
> fe:
> 
> ```js
> data.windows.reverse.sort((a, b) => {
>   if (a.new > b.new) return 1;
>   [...]
> }
> ```



> Is this to sort news in home page, depending on their status?
> 
> I think it'd be way more readable using [`Array.sort()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) with an explicit sort function.
> 
> fe:
> 
> ```js
> data.windows.reverse.sort((a, b) => {
>   if (a.new > b.new) return 1;
>   [...]
> }
> ```

done

File: src/layouts/IndexPage.jsx [hunk 1]
```diff
@@ -42,12 +43,19 @@ export const IndexPage = () => {
...
               if (daysFromStart < 14) {
                 news.new = true;
+              } else {
+                news.new = false;
               }
               if (daysToEnd < 14) {
                 news.expiring = true;
+              } else {
+                news.expiring = false;
               }
```
Comment (bfabio) [inline src/layouts/IndexPage.jsx:53]:
```suggestion
              news.expiring = daysToEnd < 14;;
```

File: src/layouts/IndexPage.jsx [hunk 2]
```diff
@@ -42,12 +43,19 @@ export const IndexPage = () => {
...
             });
-            setNewsList(data.windows.reverse().slice(0, 7));
+            const reverseNewsArr = data.windows.reverse();
+            orderedNews = reverseNewsArr.sort((a, b) => Number(b.new) - Number(a.new));
+            orderedNews = reverseNewsArr.sort((a, b) => Number(b.expiring) - Number(a.expiring));
+            setNewsList(orderedNews);
```
[the remark this answers is not in the document]
Comment (emnlpstr) [reply src/layouts/IndexPage.jsx]: I did it according to your suggestions, thanks

[the remark this answers is not in the document]
Comment (bfabio) [reply src/layouts/IndexPage.jsx]: @emnlpstr thanks, looks good :+1:

**Pull request merged**